.github/workflows/pythonpackage-linux.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install urllib3 certifi pytz pyflakes faker nose
+    - name: Generate TLS certificate
+      run: |
+        mkdir -p $HOME/.minio/certs/
```

Comment:
This is auto-created I am not sure we need to create this. Also lets use a non-homedir such that we don't need to rely on homedir location perhaps pass `--certs-dir` to MinIO ?